application/src/test/java/org/opentripplanner/ConstantsForTests.java

```diff
@@ -167,7 +171,15 @@ public static TestOtpModel buildNewPortlandGraph(boolean withElevation) {
 
       addPortlandVehicleRentals(graph);
 
-      timetableRepository.index();
+      new DirectTransferGenerator(
+        graph,
+        timetableRepository,
+        new DefaultDataImportIssueStore(),
```

Comment:
No that I would make a huge difference but I think we want the NOOP implementation here